### PR TITLE
improve .gitignore

### PR DESCRIPTION
**Pull request by chriswedgwood**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,121 @@ dist
...
+.idea/dictionaries 
+.idea/**/dataSources/ 
+.idea/**/dataSources.ids 
+.idea/**/dataSources.xml 
+.idea/**/dataSources.local.xml 
+.idea/**/sqlDataSources.xml 
+.idea/**/dynamic.xml 
+.idea/**/uiDesigner.xml 
+.idea/**/gradle.xml 
+.idea/**/libraries
```
Comment (carltongibson) [inline .gitignore:39]: Just the whole `.idea` folder?

File: .gitignore [hunk 2]
```diff
@@ -11,3 +11,121 @@ dist
...
+Control.ca-bundle 
+Control.system-ca-bundle 
+GitHub.sublime-settings 
+
+# Visual Studio Code # 
+.vscode/* 
+!.vscode/settings.json 
+!.vscode/tasks.json 
+!.vscode/launch.json 
+!.vscode/extensions.json
```
Comment (carltongibson) [inline .gitignore:130]: The whole `.vscode`?

**Comment by felixxm** [comment]
I don't see much value in ignoring IDE-specific files. In Django we decided to add a note to the `.gitignore`:
```
# If you need to exclude files such as those generated by an IDE, use
# $GIT_DIR/info/exclude or the core.excludesFile configuration variable as
# described in https://git-scm.com/docs/gitignore
```

**Comment by chriswedgwood** [comment]
> This is a generic ignore yes? Do we have half these files? Any benefit in being more specific?
> 
> OK though, as you think. 🙂
> 
> Thanks

Yes it is a generic file but I'm happy to trim. My current pain point is I keep my venv in the root of the project so I just want to exclude that. Keep things simple and add only that? 

What is best practice in this area?

**Comment by carltongibson** [comment]
I'd either try the exclude option — so add venv to `git/info/exclude`, then we don't need this at all — or add just `venv` yes.

**Comment by chriswedgwood** [comment]
Was not aware of the git/info/exclude will close the PR. Thank you both!

**Pull request closed without merging**